plugins/quetz_mamba_solve/requirements.txt

```diff
@@ -1,2 +1 @@
-mamba
```

Comment:
hmm, I think we could keep this here since it's used when doing `quetz plugin install quetz_mamba_solve`.

Although we could maybe refactor this across the board and use a different filename.